src/components/inputsList/InputsList.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import Input from '../input/Input';
+
+const InputsList = ({ inputs }) => (
+    <table>
+        <thead>
+        <tr>
+            <th>protocol</th>
+            <th>format</th>
+            <th>url</th>
+            <th>parameters</th>
+            <th>checkEvery</th>
+            <th>state</th>
+            <th>mapping</th>
+        </tr>
+        </thead>
+        <tbody>
+        {
+            renderTableBody(inputs)
+        }
+        </tbody>
+
+    </table>
+);
+
+export default InputsList;
+
+const renderTableBody = (inputs) => {
```

Comment:
1) it is better to use function then lambda function, cause function is hoisted instead of lamda
2) you don't need this function at all cause it aim is obvious, just inline it's body into table